### ISSUE TITLE
a11y: Color Contrast - Home Links

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13708]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
- Remove Button (available after activating Add Link button) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1
Main Content 
-Apply Button 

Foreground Text Color: #3D98D3
Background Text Color: #E2F0F8
Ratio: 2.72:1
Main Content 
- Add Link Button 

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF
Ratio: 1.78:1
Main Content 
- e.g. foo and e.g. bar (available after activating Add Link button)

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Apply button (on focus) 

Foreground Text Color: #FFFFFF
Background Text Color: #5D995D
Ratio: 3.39:1 
Main Content
- Applied Button  (available after activating Apply Button) "

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT.

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
>
> Instances:
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF
> Ratio: 1.78:1
> Main Content 
> - e.g. foo and e.g. bar (available after activating Add Link button)
>
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 241
